modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -450,6 +450,12 @@ export default class I3SConverter {
       this.options.maxDepth
     );
 
+    this.layers0!.attributeStorageInfo = this.attributeMetadataInfo.attributeStorageInfo;
+    this.layers0!.fields = this.attributeMetadataInfo.fields;
+    this.layers0!.popupInfo = this.attributeMetadataInfo.popupInfo;
+
+    this.layers0!.layerType = _3D_OBJECT_LAYER_TYPE;
```

Comment:
It seems we never create "Integrated mesh". The idea was when we don't have feature segmentation, we create "Integrated mesh" layer type. Now it always "3D objects".
Test it with "Frankfurt" tileset, for example

Reply:
Done. All these set operations will be performed if there is at least one attribute.